examples/websocket-client-self-signed-jwt-auth/websocket_client_self_signed_jwt_auth.md

```diff
@@ -1,6 +1,6 @@
 # WebSocket client - Self signed JWT authentication
 
-A client, which is secured with self-signed JWT can be used to connect to a secured service. The client is enriched with the `Authorization: Bearer <token>` header by passing the `websocket:JwtIssuerConfig` to the `auth` configuration of the client. A self-signed JWT is issued before the request is sent.
+The `websocket:Client` can connect to a service secured with self-signed JWT, by enriching the initial HTTP request from the client with the `Authorization: Bearer <token>` header. The required configurations for this can be specified in the `auth` field of the client configuration. A self-signed JWT is issued before the request is sent.
```

Comment:
```suggestion
The `websocket:Client` can connect to a service secured with self-signed JWT by enriching the initial HTTP request from the client with the `Authorization: Bearer <token>` header. The required configurations for this can be specified in the `auth` field of the client configuration. A self-signed JWT is issued before the request is sent.
```